jwk_test.go

```diff
@@ -354,13 +354,40 @@ func TestInvalidThumbprintsX509(t *testing.T) {
 
 	var jwk2 JSONWebKey
 	err = jwk2.UnmarshalJSON(mismatchedThumbprints)
-	if err == nil {
-		t.Error("should not unmarshal JWK with mismatched thumbprints")
+	if err != nil {
```

Comment:
this change (and the one below) is only for debugging. Turns out that the tests are passing, because there's another error in the generated JWK (I was expecting only thumbprints to cause errors)